ios/FluentUI.Demo/FluentUI.Demo/DemoListViewController.swift

```diff
@@ -71,7 +92,11 @@ class DemoListViewController: UITableViewController {
         let demoController = demo.controllerClass.init(nibName: nil, bundle: nil)
         demoController.title = demo.title
         navigationController?.pushViewController(demoController, animated: true)
+
+        UserDefaults.standard.set(demo.title, forKey: DemoListViewController.lastDemoControllerKey)
     }
 
     let cellReuseIdentifier: String = "TableViewCell"
+    private var didAutoLaunch: Bool = false
```

Comment:
nit:
This variable tracks the whether or not it's the first launch... not if the auto launch of a view controller happened.
Can we rename it to isFirstLaunch and default it to true (inverting the logic on viewDidAppear)?